README.md

```diff
@@ -30,26 +30,26 @@ YAML-compatible mustache implementation such as `ruby-mustache`
 or `mustache-go` (available as a
 [Nix](https://nixos.org/nixos/packages.html?attr=mustache-go&channel=nixpkgs-unstable) package).
 
-To enable generating files using one such mustache tool, you should
-thus write a `meta.yml` file containing the required values. For
-example, assuming the templates are available in a sibling directory,
-configuration and boilerplate files in the
-[AAC Tactics](https://github.com/coq-community/aac-tactics)
-project are generated as follows:
-```shell
-mustache meta.yml ../templates/README.md.mustache > README.md
-mustache meta.yml ../templates/coq.opam.mustache > coq-aac-tactics.opam
-mustache meta.yml ../templates/default.nix.mustache > default.nix
-mustache meta.yml ../templates/.travis.yml.mustache > .travis.yml
-```
-Other projects using the templates in a similar way include
-[Chapar](https://github.com/coq-community/chapar) and
-[Lemma Overloading](https://github.com/coq-community/lemma-overloading).
+To enable generating files (e.g, `.opam` files and CI setup) using one such
+mustache tool, one needs to provide a `meta.yml` file containing the required
+values. Depending on your project and the desired setup, the `meta.yml` files of
+one of the following projects may serve as a starting point:
+
+- [Huffman](https://github.com/coq-community/huffman) (pure library, no dependencies, coq-community) :
+  [meta.yml](https://github.com/coq-community/huffman/blob/master/meta.yml)
+- [AAC_tactics](https://github.com/coq-community/aac-tactics) (plugin, one-branch per Coq version):
+  [meta.yml](https://github.com/coq-community/aac-tactics/blob/master/meta.yml).
+- [reglang](https://github.com/coq-community/reglang) (pure library, mathcomp, custom docker images) :
+  [meta.yml](https://github.com/coq-community/reglang/blob/master/meta.yml).
+- [StructTact](https://github.com/uwplse/StructTact) (pure library, no dependencies, no coq-community):
+  [meta.yml](https://github.com/uwplse/StructTact/blob/master/meta.yml)
```

Comment:
```suggestion
- [Huffman](https://github.com/coq-community/huffman) (pure library, no dependencies, coq-community):
  [meta.yml](https://github.com/coq-community/huffman/blob/master/meta.yml)
- [AAC Tactics](https://github.com/coq-community/aac-tactics) (plugin, one-branch per Coq version):
  [meta.yml](https://github.com/coq-community/aac-tactics/blob/master/meta.yml)
- [reglang](https://github.com/coq-community/reglang) (pure library, mathcomp, custom docker images):
  [meta.yml](https://github.com/coq-community/reglang/blob/master/meta.yml)
- [StructTact](https://github.com/uwplse/StructTact) (pure library, no dependencies, no coq-community):
  [meta.yml](https://github.com/uwplse/StructTact/blob/master/meta.yml)
```
Nitpicks.

Reply:
Nitpicking is accepted, even welcome, in this context.